ProductService/src/main/resources/application-local.yaml

```
@@ -31,6 +31,17 @@ spring:
+    optimization: false
+
+exchange-rate:
+  file-name: exchange-rate.json
+
+
+currency-service:
+  host: http://localhost:8080
+  timeout: 1000
+  retry: 3
+  methods:
+    get-currency: /api/v1/currency
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)